ui/src/app/utils/subscribable.ts

```diff
@@ -45,18 +45,14 @@ export function subscribable<T>(): Subscribable<T> {
       };
     },
     next: (newValue?: T, oldValue?: T) => {
-      fp.forEach(
-        (fn: subscriber) =>
-          setTimeout(() => {
-            try {
-              fn(newValue, oldValue);
-            } catch (e) {
-              // Ignore the error - ideally the supplied fn should
-              // supply its error handling
-            }
-          }, 0),
-        subscribers
-      );
+      fp.forEach((fn: subscriber) => {
+        try {
+          fn(newValue, oldValue);
+        } catch (e) {
+          // Ignore the error - ideally the supplied fn should
+          // supply its error handling
```

Comment:
I really like the narrow scope of this PR, so I would not change it, but I'd very much like to see (perhaps in a follow-up PR) what would happen if we:
```suggestion
          console.error('Uncaught error in subscriber function:', e)
```